modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-volley/build.mustache

```diff
@@ -0,0 +1,125 @@
+{{#useAndroidMavenGradlePlugin}}
+group = '{{groupId}}'
+project.version = '{{artifactVersion}}'
+{{/useAndroidMavenGradlePlugin}}
+
+buildscript {
+    repositories {
+        mavenLocal()
+        google()
+        maven {
+            url 'https://dl.google.com/dl/android/maven2'
+        }
+        jcenter()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20"
+        classpath 'com.android.tools.build:gradle:4.0.2'
+        {{#useAndroidMavenGradlePlugin}}
+        classpath 'com.github.dcendents:android-maven-gradle-plugin:1.5'
+        {{/useAndroidMavenGradlePlugin}}
+    }
+}
+
+allprojects {
+    repositories {
+        google()
+        jcenter()
+    }
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+{{#useAndroidMavenGradlePlugin}}
+apply plugin: 'com.github.dcendents.android-maven'
+{{/useAndroidMavenGradlePlugin}}
+
+android {
+    compileSdkVersion 29
```

Comment:
@alisters Currently the minimum sdk version required is 30, so could you please update this and algo `targetSdkVersion`?

Reply:
Very light in experience with Android. Does this look correct ?
compileSdkVersion 30
    defaultConfig {
        minSdkVersion 21
        targetSdkVersion 30
    }

Reply:
Looks good to me 👍